examples/databricks_agent/databricks_agent/databricks_agent_example_usage.py

```diff
@@ -11,7 +11,7 @@
 
 import flytekit
 from flytekit import Resources, task, workflow
-from flytekitplugins.spark import Databricks
+from flytekitplugins.spark import DatabricksV2
```

Comment:
nit: Users don't need to change their task config if they do this. It's useful if they have multiple databricks tasks.
```suggestion
from flytekitplugins.spark import DatabricksV2 as Databricks
```